ocn.ne.jp,"HTTPSConnectionPool(host='www.ocn.ne.jp', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='www.ocn.ne.jp', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
odisseias.com,"HTTPSConnectionPool(host='odisseias.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1131)')))","HTTPSConnectionPool(host='www.odisseias.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1131)')))"
odkazprestarostu.sk,"HTTPSConnectionPool(host='www.odkazprestarostu.sk', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.odkazprestarostu.sk', port=443): Read timed out. (read timeout=2)"
oeticket.com,"HTTPSConnectionPool(host='oeticket.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f987d6e2af0>, 'Connection to oeticket.com timed out. (connect timeout=2)'))","HTTPConnectionPool(host='www.oeticket.com', port=80): Read timed out. (read timeout=2)"
offen.net,"HTTPSConnectionPool(host='offen.net', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='www.offen.net', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
office1.bg,"HTTPSConnectionPool(host='www.office1.bg', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.office1.bg', port=443): Read timed out. (read timeout=2)"
officedepot.co.cr,"HTTPSConnectionPool(host='officedepot.co.cr', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='officedepot.co.cr', port=80): Read timed out. (read timeout=2)"
officedepot.com.sv,"HTTPSConnectionPool(host='officedepot.com.sv', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f987e9b5af0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='officedepot.com.sv', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f987e43f550>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
officedepot.eu,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","HTTPConnectionPool(host='officedepot.eu', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f987e30b490>, 'Connection to officedepot.eu timed out. (connect timeout=2)'))"
offipalsta.com,"HTTPSConnectionPool(host='offipalsta.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='offipalsta.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
ofppt.ma,"HTTPSConnectionPool(host='ofppt.ma', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f987ea04fa0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='ofppt.ma', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f987d024400>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
ohchr.org,"HTTPSConnectionPool(host='ohchr.org', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1131)')))","HTTPSConnectionPool(host='ohchr.org', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1131)')))"
oic.or.th,"HTTPSConnectionPool(host='oic.or.th', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f987f77a2e0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='oic.or.th', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f987c2b92e0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
oj.gob.gt,"HTTPSConnectionPool(host='oj.gob.gt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f987beeaac0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='oj.gob.gt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f987d58fb20>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
okanhastanesi.com.tr,"HTTPSConnectionPool(host='okanhastanesi.com.tr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='okanhastanesi.com.tr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
okmagazine.ro,"HTTPSConnectionPool(host='okmagazine.ro', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='okmagazine.ro', port=443): Read timed out. (read timeout=2)"
okulhaberleri.net,"HTTPSConnectionPool(host='www.okulhaberleri.net', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.okulhaberleri.net', port=443): Read timed out. (read timeout=2)"
oliberal.com,"HTTPSConnectionPool(host='oliberal.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='oliberal.com', port=443): Read timed out. (read timeout=2)"
olivemagazine.gr,"HTTPSConnectionPool(host='olivemagazine.gr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1131)')))","HTTPSConnectionPool(host='olivemagazine.gr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1131)')))"
olx.co.id,"HTTPSConnectionPool(host='olx.co.id', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f987c6e8ee0>, 'Connection to olx.co.id timed out. (connect timeout=2)'))","HTTPConnectionPool(host='www.olx.co.id', port=80): Read timed out. (read timeout=2)"
olx.co.za,"HTTPSConnectionPool(host='olx.co.za', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f987d20ab20>, 'Connection to olx.co.za timed out. (connect timeout=2)'))","HTTPSConnectionPool(host='www.olx.co.za', port=443): Read timed out. (read timeout=2)"
olx.in,"HTTPSConnectionPool(host='olx.in', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f987e7015e0>, 'Connection to olx.in timed out. (connect timeout=2)'))","HTTPConnectionPool(host='www.olx.in', port=80): Read timed out. (read timeout=2)"
omed.bg,"HTTPSConnectionPool(host='omed.bg', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.omed.bg', port=443): Read timed out. (read timeout=2)"
omegalive.com.cy,"HTTPSConnectionPool(host='omegalive.com.cy', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='omegalive.com.cy', port=443): Read timed out. (read timeout=2)"
omelhordobairro.com,"HTTPSConnectionPool(host='omelhordobairro.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='www.omelhordobairro.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
omni7.jp,"HTTPSConnectionPool(host='omni7.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f987f5bcb20>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='omni7.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f987e41cb80>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
omron.co.jp,"HTTPSConnectionPool(host='omron.co.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f987ee4bdf0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='omron.co.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f987bfa0910>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
oncyprus.com,"HTTPSConnectionPool(host='www.oncyprus.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.oncyprus.com', port=443): Read timed out. (read timeout=2)"
onep.go.th,"HTTPSConnectionPool(host='onep.go.th', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f987ee38430>, 'Connection to onep.go.th timed out. (connect timeout=2)'))","HTTPConnectionPool(host='onep.go.th', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f987d487040>, 'Connection to onep.go.th timed out. (connect timeout=2)'))"
onepagefree.com,"HTTPSConnectionPool(host='onepagefree.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: UNSUPPORTED_PROTOCOL] unsupported protocol (_ssl.c:1131)')))","HTTPConnectionPool(host='onepagefree.com', port=80): Read timed out. (read timeout=2)"
onesearch.id,"HTTPSConnectionPool(host='onesearch.id', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='onesearch.id', port=443): Read timed out. (read timeout=2)"
onko-i.si,"HTTPSConnectionPool(host='onko-i.si', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f987e3159d0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='onko-i.si', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f987a37c550>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
onlineexpo.com,"HTTPSConnectionPool(host='onlineexpo.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='onlineexpo.com', port=443): Read timed out. (read timeout=2)"
op.se,"HTTPSConnectionPool(host='op.se', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='www.op.se', port=443): Read timed out. (read timeout=2)"
opais.co.ao,"HTTPSConnectionPool(host='opais.co.ao', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='opais.co.ao', port=443): Read timed out. (read timeout=2)"
openingsuren.info,"HTTPSConnectionPool(host='www.openingsuren.info', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.openingsuren.info', port=443): Read timed out. (read timeout=2)"
opentextbc.ca,"HTTPSConnectionPool(host='opentextbc.ca', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1131)')))","HTTPSConnectionPool(host='opentextbc.ca', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1131)')))"
opsmoac.go.th,"HTTPSConnectionPool(host='opsmoac.go.th', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f987e258580>, 'Connection to opsmoac.go.th timed out. (connect timeout=2)'))","HTTPConnectionPool(host='opsmoac.go.th', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f987e797610>, 'Connection to opsmoac.go.th timed out. (connect timeout=2)'))"
optus.com.au,"HTTPSConnectionPool(host='www.optus.com.au', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='www.optus.com.au', port=80): Read timed out. (read timeout=2)"
orange.ma,"HTTPSConnectionPool(host='orange.ma', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1131)')))","HTTPSConnectionPool(host='orange.ma', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1131)')))"
ordenjuridico.gob.mx,"HTTPSConnectionPool(host='ordenjuridico.gob.mx', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f987e12bc40>: Failed to establish a new connection: [Errno 111] Connection refused'))","HTTPConnectionPool(host='ordenjuridico.gob.mx', port=80): Read timed out. (read timeout=2)"
ore.edu.pl,"HTTPSConnectionPool(host='ore.edu.pl', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='ore.edu.pl', port=443): Max retries exceeded with url: // (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
orebro.se,"HTTPSConnectionPool(host='www.orebro.se', port=443): Read timed out.","HTTPSConnectionPool(host='www.orebro.se', port=443): Read timed out."
oreillyauto.com,"HTTPSConnectionPool(host='www.oreillyauto.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.oreillyauto.com', port=443): Read timed out. (read timeout=2)"
organiseereentopevent.nl,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
orientini.com,"HTTPSConnectionPool(host='orientini.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f987b9e99a0>, 'Connection to orientini.com timed out. (connect timeout=2)'))","HTTPConnectionPool(host='orientini.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f987d79aeb0>, 'Connection to orientini.com timed out. (connect timeout=2)'))"
orst.go.th,"HTTPSConnectionPool(host='orst.go.th', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f987a2b5520>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='orst.go.th', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f987b46efd0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
osaka-u.ac.jp,"HTTPSConnectionPool(host='osaka-u.ac.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f987e2d9820>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='osaka-u.ac.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f987ec56400>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
osaka.lg.jp,"HTTPSConnectionPool(host='osaka.lg.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f987aa5baf0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='osaka.lg.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f98779f3b50>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
ose.com.uy,"HTTPSConnectionPool(host='ose.com.uy', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1131)')))","HTTPConnectionPool(host='ose.com.uy', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f987cd0f370>, 'Connection to ose.com.uy timed out. (connect timeout=2)'))"
osloskolen.no,"HTTPSConnectionPool(host='osloskolen.no', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f987e43fd00>, 'Connection to osloskolen.no timed out. (connect timeout=2)'))","HTTPConnectionPool(host='osloskolen.no', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f987e655730>, 'Connection to osloskolen.no timed out. (connect timeout=2)'))"
othaimmarkets.com,"HTTPSConnectionPool(host='othaimmarkets.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='www.othaimmarkets.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
otofun.net,"HTTPSConnectionPool(host='www.otofun.net', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.otofun.net', port=443): Read timed out. (read timeout=2)"
otpbanka.hr,"HTTPSConnectionPool(host='otpbanka.hr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f987e1d2190>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='otpbanka.hr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f987a31e7f0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
otzovik.com,"HTTPSConnectionPool(host='otzovik.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f987b452e50>, 'Connection to otzovik.com timed out. (connect timeout=2)'))","HTTPConnectionPool(host='otzovik.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f987f48b820>, 'Connection to otzovik.com timed out. (connect timeout=2)'))"
ous-hf.no,"HTTPSConnectionPool(host='ous-hf.no', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f987d6c4250>, 'Connection to ous-hf.no timed out. (connect timeout=2)'))","HTTPConnectionPool(host='ous-hf.no', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f98777c88b0>, 'Connection to ous-hf.no timed out. (connect timeout=2)'))"
overclockers.ru,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
overstock.ca,"HTTPSConnectionPool(host='www.overstock.ca', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.overstock.ca', port=443): Read timed out. (read timeout=2)"
overstock.com,"HTTPSConnectionPool(host='www.overstock.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.overstock.com', port=443): Read timed out. (read timeout=2)"
ovt.lv,"HTTPSConnectionPool(host='ovt.lv', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'ovt.lv' doesn't match either of '*.datateks.lv', 'datateks.lv'"")))","HTTPConnectionPool(host='ovt.lv', port=80): Read timed out. (read timeout=2)"
owlapps.net,"HTTPSConnectionPool(host='owlapps.net', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='www.owlapps.net', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
oxfordlearnersdictionaries.com,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
oxfordreference.com,"HTTPSConnectionPool(host='oxfordreference.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='oxfordreference.com', port=443): Read timed out. (read timeout=2)"
ozs.si,"HTTPSConnectionPool(host='ozs.si', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f987b65ddf0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='ozs.si', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f9878a0ae50>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
p-world.co.jp,"HTTPSConnectionPool(host='p-world.co.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f98788b4bb0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='p-world.co.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f987ef13340>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
paagman.nl,"HTTPSConnectionPool(host='www.paagman.nl', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.paagman.nl', port=443): Read timed out. (read timeout=2)"
pagesjaunes.ca,"HTTPSConnectionPool(host='www.pagesjaunes.ca', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1131)')))","HTTPSConnectionPool(host='www.pagesjaunes.ca', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1131)')))"
pai-hang-bang.com,"HTTPSConnectionPool(host='pai-hang-bang.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='pai-hang-bang.com', port=443): Read timed out. (read timeout=2)"
painaidii.com,"HTTPSConnectionPool(host='painaidii.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPConnectionPool(host='www.painaidii.com', port=80): Read timed out. (read timeout=2)"
paintandpowerwashct.com,"HTTPSConnectionPool(host='paintandpowerwashct.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f98765e4b20>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='paintandpowerwashct.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f987a2cb520>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
pal-system.co.jp,"HTTPSConnectionPool(host='pal-system.co.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f987ce1fc10>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='pal-system.co.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f987c5e80a0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
palangostiltas.lt,"HTTPSConnectionPool(host='www.palangostiltas.lt', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.palangostiltas.lt', port=443): Read timed out. (read timeout=2)"
palmers.co.nz,"HTTPSConnectionPool(host='www.palmers.co.nz', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.palmers.co.nz', port=443): Read timed out. (read timeout=2)"
pamiatky.sk,"HTTPSConnectionPool(host='pamiatky.sk', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f987db66c10>, 'Connection to pamiatky.sk timed out. (connect timeout=2)'))","HTTPConnectionPool(host='pamiatky.sk', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f987b344df0>, 'Connection to pamiatky.sk timed out. (connect timeout=2)'))"
panspc.lt,"HTTPSConnectionPool(host='panspc.lt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f987ccdc610>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='panspc.lt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f987f5763a0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
paraguay.com,"HTTPSConnectionPool(host='paraguay.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1131)')))","HTTPSConnectionPool(host='www.paraguay.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1131)')))"
parisinfo.com,"HTTPSConnectionPool(host='parisinfo.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1131)')))","HTTPSConnectionPool(host='parisinfo.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1131)')))"
parks.it,"HTTPSConnectionPool(host='parks.it', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f9878a0a340>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='parks.it', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f987e8c4f10>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
parlament.ch,Exceeded 30 redirects.,Exceeded 30 redirects.
parlament.gov.rs,"HTTPSConnectionPool(host='parlament.gov.rs', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f987ed40e20>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='parlament.gov.rs', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f9879a8cf10>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
parliament.go.th,"HTTPSConnectionPool(host='parliament.go.th', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f987aa4f3d0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='parliament.go.th', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f987ae33c10>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
parliament.lk,"HTTPSConnectionPool(host='parliament.lk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1131)')))","HTTPSConnectionPool(host='parliament.lk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1131)')))"
parp.gov.pl,"HTTPSConnectionPool(host='parp.gov.pl', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='parp.gov.pl', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
partyalert.nl,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
pasgo.vn,"HTTPSConnectionPool(host='pasgo.vn', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='pasgo.vn', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
paslaugos24.lt,"HTTPSConnectionPool(host='paslaugos24.lt', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='www.paslaugos24.lt', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
